### PR TITLE
[FEAT] Fetch user personal information

### PR DESCRIPTION
**Pull request by g-ongenae**
## Description

- Fetch user personal information from the user connection (`item`) and add the `owner.name` to each account based on its connection (`item.id` / `item_id`).

#### Note

- Requires https://github.com/algoan/rest-nodejs/pull/34

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by meriamBenSassi — APPROVED** [review]
LGTM !

**Review by fabong — APPROVED** [review]
LGTM

File: src/hooks/services/hooks.service.ts [hunk 1]
```diff
@@ -197,8 +202,18 @@ export class HooksService {
...
       accounts,
     });
+
+    /**
+     * 2.b. Get personal information
+     */
+    const userInfo: BridgeUserInformation[] = await this.aggregator.getUserPersonalInformation(
+      accessToken,
+      serviceAccount.config as ClientConfig,
+    );
```
Comment (ccoeurderoy) [inline src/hooks/services/hooks.service.ts:212]: Add a try-catch statement to no block the aggregation process if it fails

**Review by ccoeurderoy — CHANGES_REQUESTED** [review]
LGTM

**Review by ccoeurderoy — APPROVED** [review]
👌 thank you @g-ongenae!

**Pull request merged**